pkg/scheduler/util/test_utils.go

```diff
@@ -256,18 +256,44 @@ func BuildPriorityClass(name string, value int32) *schedulingv1.PriorityClass {
 
 // FakeBinder is used as fake binder
 type FakeBinder struct {
-	sync.Mutex
-	Binds   map[string]string
+	sync.RWMutex
+	binds   map[string]string
 	Channel chan string
 }
 
+// NewFakeBinder returns a instance of FakeBinder
+func NewFakeBinder(buffer int) *FakeBinder {
```

Comment:
The FakeBinder is exposed to the public, is this necessary?

Reply:
Currently it is used in uthelper package: `binder := test.binder.(*util.FakeBinder)`.